perception/occupancy_map_monitor/include/moveit/occupancy_map_monitor/occupancy_map_monitor.h

```diff
@@ -122,6 +122,12 @@ class OccupancyMapMonitor
     return active_;
   }
 
+  /** \brief Enable sending diffs of the occupancy map */
+  void enableOctomapDiff(bool enable) { octomap_diff_enable = enable; }
```

Comment:
Should this function be named `setOctomapDiff(bool enable)`?
Personally, I find `enableXYZ(false);` calls a bit weird. No hard feelings though.